lib/src/utils/uuid_shortener.dart

```diff
@@ -22,14 +24,31 @@ class UuidShortener {
 
   int get _base => alphabet.length;
 
+  /// Whether a string is a valid short string within the [alphabet].
```

Comment:
Che cosa fa?

Reply:
controlla se la string in input è valida nell’alfabeto corrente andando a vedere se ogni carattere della stringa appartiene all’alfabeto